.travis.yml

```diff
@@ -3,20 +3,22 @@ compiler:
   - clang
 sudo: false
 cache: ccache
+os: linux
 matrix:
   include:
-    - os: linux
+    - name: "Linter"
```

Comment:
TIL :)

Reply:
I believe it's new https://changelog.travis-ci.com/announcing-build-stages-ga-job-names-and-conditions-v1-as-platform-default-66397